Number_types/test/Number_types/utilities.cpp

```diff
@@ -1,10 +1,5 @@
 #include <CGAL/config.h>
 
-// TODO: solve conflict of CORE with GMPXX
-#ifdef CGAL_USE_CORE
-#undef CGAL_USE_GMPXX
-#endif
```

Comment:
This is an old (2007) commit and I did not get any error from removing this, does this ring a bell to anyone?